docs/docs/dev/architecture/event-sourcing.md

```diff
@@ -1,11 +1,9 @@
-## Event sourcing
+# Event sourcing
 
-The `hydra-node` is an event sourced application. This means that the main logic is processing _inputs_ (also called commands) and produces _events_. These events are saved and loaded to persist application state across restarts. Also, most events are transformed to _outputs_ and can be observed on the `hydra-node` API.
+The `hydra-node` is an event sourced application. This means that the main logic is processing _inputs_ (also called commands) and produces _events_. These events are saved and loaded to persist application state across restarts. Also, most events are transformed to _outputs_ and can be observed on the `hydra-node` API. See 
```

Comment:
Looks like there is a `See` in the end of the sentence that doesn't belong there.